hw-tictac/src/components/proofOfConcept/ProofOfConcept.jsx

```diff
@@ -2,26 +2,22 @@ import React from "react";
 import { Box, Typography } from "@mui/material";
 
 export function ProofOfConcept() {
-
     return (
         <Box display="flex" flexDirection="column" gap={2}>
-            <Typography variant="h5">
-                AliceNet Tictactoe (POC)
-            </Typography>
+            <Typography variant="h5">AliceNet Tictactoe (POC)</Typography>
             <Box display="flex" flexDirection="row" gap={2}>
                 <Typography variant="span" fontFamily="Roboto">
-                    "This application is to demonstrate the use of multi-signature wallets where two wallets are coming
+                    This application is to demonstrate the use of multi-signature wallets where two wallets are coming
```

Comment:
In my defense, I just copied whatever it was in the Prototype :P

Reply:
Defense accepted. :joy_cat: